include/clspv/Compiler.h

```diff
@@ -74,22 +90,27 @@ extern "C" {
 // |options|            - String of options to pass to Clspv compiler.
 // |output_binary|      - Handle to compiler output/result.
 // |output_binary_size| - Size of compiler output/result (in bytes).
-// |output_log|         - Handle to compiler build log.
+// |output_log|         - Handle to compiler build log,
+// 			  will assume log is not needed if param is NULL.
+// 			  its value will be NULL when no build log is empty.
```

Comment:
It's not clear to me what the precision you are trying to add here.
Maybe something like that?
```suggestion
// |output_log|         - Handle to compiler build log. `output_log` can be NULL
//                        in which case no log will be generated.
```

Reply:
This seems much better and make sense. Many thanks for pointing me this.